### PR TITLE
powsybl-ws-dependencies 2.7.0

### PR DESCRIPTION
**Pull request by antoinebhs**

### CODE REVIEW

File: src/main/java/com/powsybl/network/conversion/server/GeneratorActivePowerControlSerDe.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import com.powsybl.commons.extensions.ExtensionSerDe;
+import com.powsybl.commons.io.DeserializerContext;
+import com.powsybl.commons.io.SerializerContext;
+import com.powsybl.commons.io.TreeDataReader;
+import com.powsybl.iidm.network.Injection;
+import com.powsybl.iidm.network.extensions.ActivePowerControl;
+import com.powsybl.iidm.network.extensions.ActivePowerControlAdder;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
```
Comment (antoinebhs) [inline src/main/java/com/powsybl/network/conversion/server/GeneratorActivePowerControlSerDe.java:20]: REMOVE HACK comment?

File: src/main/java/com/powsybl/network/conversion/server/GeneratorActivePowerControlSerDe.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ */
+@AutoService(ExtensionSerDe.class)
+public class GeneratorActivePowerControlSerDe<T extends Injection<T>> extends AbstractExtensionSerDe<T, ActivePowerControl<T>> {
```
Comment (etiennehomer) [inline src/main/java/com/powsybl/network/conversion/server/GeneratorActivePowerControlSerDe.java:23]: You should keep a comment saying that this class should be deleted later

**Review by etiennehomer — CHANGES_REQUESTED** [review]

**Review by etiennehomer — APPROVED** [review]

**Comment by antoinebhs** [comment]
Find a comment to put on the duplicated classes from rte-core

File: src/main/java/com/powsybl/network/conversion/server/GeneratorActivePowerControlSerDe.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    public GeneratorActivePowerControlSerDe() {
+        super("generatorActivePowerControl", "network", ActivePowerControl.class, "generatorActivePowerControl.xsd", "http://www.itesla_project.eu/schema/iidm/ext/generator_active_power_control/1_0", "gapc");
+    }
+
+    @Override
+    public void write(ActivePowerControl activePowerControl, SerializerContext serializerContext) {
+        throw new IllegalStateException("Should never be called");
```
[the remark this answers is not in the document]
Comment (antoinebhs) [reply src/main/java/com/powsybl/network/conversion/server/GeneratorActivePowerControlSerDe.java:33]: I agree with you but these 3 classes (GeneratorActivePowerControlSerDe.java, GeneratorShortCircuitsSerDe.java,VoltageLevelShortCircuitsSerDe.java) are duplicated from rte-core/iidm-cvg-extensions. I added a comment at the top of the file to make it clearer.
I think we should not change the message to be consistent with the private repository rte-core/iidm-cvg-extensions.

**Review by AbdelHedhili — APPROVED** [review]

**Pull request merged**